.changeset/itchy-timers-lick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-docs/gatsby-theme-docs': patch
```

Comment:
We are introducing a new feature now (the config), so this should be a minor bump.

```suggestion
'@commercetools-docs/gatsby-theme-docs': minor
```